modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbDataRegion.java

```diff
@@ -34,7 +33,7 @@
  */
 public class RocksDbDataRegion {
     /** Region configuration. */
-    private final RocksDbDataRegionConfiguration cfg;
+    private final RocksDbDataRegionView cfg;
```

Comment:
I think that the name is not very suitable, maybe `configView`?

Reply:
fixed